### PR TITLE
[ONNX] Extend `test_fx_op_consistency.py` to cover ExportedProgram model type

### PR DESCRIPTION
**Pull request by titaiwangms**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #114886

This PR covers `ExportedProgram` to `test_fx_op_consistency.py`, which helps us identify the necessary but missing io_steps.
Next, we should refactor the tests to actually cover all ops supported by registry.

### CODE REVIEW

**Comment by titaiwangms** [comment]
Blocked by #114892

File: test/onnx/onnx_test_common.py [hunk 1]
```diff
@@ -520,6 +519,7 @@ class DecorateMeta:
         test_behavior: The behavior of the test case. [skip or xfail]
         matcher: The matcher to apply to the test case.
         enabled_if: Whether to enable test behavior. Usually used on onnx/ort version control
+        torchmodeltype: The type of the torch model. Defaults to None.
```
Comment (thiagocrepaldi) [inline test/onnx/onnx_test_common.py:522]:
```suggestion
        torchmodeltype: The type of the torch model. Defaults to None.
```
At `test_fx_to_onnx_with_onnxruntime.py` this argument is named `model_type`. We should use the same name to make it uniform across the test files

**Review by thiagocrepaldi — APPROVED** [review]
Overall, LGTM. 2 non-blocking comments below

A minor comment is to rename `torchmodeltype` to match `test_fx_to_onnx_onnxruntime.py` naming.

A second comment is that the changes at `torch/onnx/_internal/fx/torch_export_graph_extractor.py` are (nice) fixes to `TorchExport` and could have their own PR to prevent mixing test infra improvement with exporter bug fixing.

**Comment by titaiwangms** [comment]
Blocked by unknow reason of discrepancy test result between local and CI...

**Pull request closed without merging**